app/controllers/collections_controller.rb

```diff
@@ -37,7 +37,7 @@ def index
   end
 
   def show
-    redirect_to(ontology_path(id: params[:ontology_id], p: 'collections', collectionid: params[:id], lang: request_lang)) and return unless turbo_frame_request?
+    redirect_to(ontology_path(id: params[:ontology], p: 'collections', collectionid: params[:id], lang: request_lang)) and return if turbo_frame_request?
```

Comment:
here why do  change  `params[:ontology_id]` by `params[:ontology]` ?

Reply:
and it need to be `unless` not `if`

Reply:
I changed it because there is no `ontology_id` in `params` (look at the screenshot) there's only `ontology`. I don't know if the redirection ever happens because when we access by the browser directly the index method get triggered otherwise we are using a turboframe 
 
![Screenshot from 2024-09-21 08-46-09](https://github.com/user-attachments/assets/189c137d-2f38-48f0-81e6-9e2d1ae5c8f2)

Reply:
Good catch :+1: